spritz/app/src/main/res/layout/page.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why not `<View />` since it's an empty view?

PS: `RelativeLayout` doesn't look at the `orientation`

Reply:
AFAIK there's no way to unregister a `OnPageChangeListener`.

Reply:
Wrong comment?

Reply:
Done in 2be9a1807bd9569d4df9fcded758b2d4efe8dbd4.
Yes, it was a wrong comment :D